src/server/loginErrorHandle.js

```diff
@@ -0,0 +1,28 @@
+const db = require('./database');
+
+function login(req, res) {
+  res.setHeader('Content-Type', 'application/json');
+  let email = req.body.email;
+  let password = req.body.password;
+  let status = 200;
+  let result = db.checkUserInfo(email, password);
+  let contentType = req.headers['content-type'].toLowerCase();
+  let obj = {};
+  if (email === '' || password === '') {
```

Comment:
you should do the static validation first, then query the mongo. Now you send shitty data to mongo then you will return with the error. You eat a lot of resources, this is worthless